public/locales/en/p-charts.json

```diff
@@ -0,0 +1,4 @@
+{
+  "meta-title": "Iron Fish: Charts",
```

Comment:
`Iron Fish` might be strippable from some of these places, I don't know that we'll localize that. Doesn't necessarily need to be in this PR

Reply:
Fair. I might consider adding `Iron Fish` to a shared / `common` translation file.